genesyscloud/outbound/data_source_genesyscloud_outbound_messagingcampaign_test.go

```diff
@@ -84,10 +83,9 @@ func TestAccDataSourceOutboundMessagingCampaign(t *testing.T) {
 		)
 	)
 
-	config := platformclientv2.GetDefaultConfiguration()
-	err := config.AuthorizeClientCredentials(os.Getenv("GENESYSCLOUD_OAUTHCLIENT_ID"), os.Getenv("GENESYSCLOUD_OAUTHCLIENT_SECRET"))
+	config, err := gcloud.AuthorizeSdk()
```

Comment:
I made this change in a few different files. In a lot of places this `AuthorizeClientCredentials` function was being called instead of using the function `AuthorizeSdk`, which handles all the logic of grabbing the client id/secret env variables.